src/dune

```diff
@@ -0,0 +1,38 @@
+(library
+ (name elpi_plugin_pure)
+ (public_name coq-elpi)
+ (flags :standard -rectypes -warn-error -3-9-27-33)
+ (preprocess (pps ppx_deriving.show ppx_deriving.ord))
+ (libraries coq.plugins.ltac elpi))
+
+(rule
+ (targets coq_elpi_config.ml)
+ (action
+  (with-stdout-to %{targets}
+   (run ocamlfind query -format "let elpi_dir = \"%d\"" elpi))))
+
+(coq.pp (modules coq_elpi_vernacular_syntax))
+
+; We link elpi.cmxa statically into the cmxs; this will break if some
+; other plugin is linking elpi, so it should be replaced by
+; ocamlfind-based dependency loading (cf coq/coq#7698 )
```

Comment:
I'd like this link to be mentioned here:
https://dune.readthedocs.io/en/stable/advanced-topics.html#dynamic-loading-of-packages

Reply:
After a second thought this is too messy to implement with Dune + Coq Dune [due to the way .v files depend on plugin code], so I propose we add proper tracking to the plugin loader and remove all these hacks in the dune build.